src/sql/base/test/browser/ui/selectBox/selectBox.test.ts

```diff
@@ -38,7 +38,7 @@ suite('Select Box tests', () => {
 	});
 
 	test('the selected option changes', () => {
-		const sb = new SelectBox(options, options[1].value, undefined, undefined, undefined);
+		const sb = new SelectBox(options, options[1].value, undefined!, undefined!, undefined!);
```

Comment:
probably should be passing real values into here, but its a test and seems to be working as is so I'm not going to take the time to fix.